list-messages/src/main/java/no/unit/nva/pubication/messages/list/ListMessagesHandler.java

```diff
@@ -0,0 +1,60 @@
+package no.unit.nva.pubication.messages.list;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.time.Clock;
+import java.util.List;
+import no.unit.nva.publication.service.impl.MessageService;
+import no.unit.nva.publication.service.impl.ResourceMessages;
+import no.unit.nva.publication.storage.model.UserInstance;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ListMessagesHandler extends ApiGatewayHandler<Void, ResourceMessages[]> {
+
+    public static final Logger LOGGER = LoggerFactory.getLogger(ListMessagesHandler.class);
+    private final MessageService messageService;
+
+    @JacocoGenerated
+    public ListMessagesHandler() {
+        this(new Environment(), defaultMessageService());
+    }
+
+    public ListMessagesHandler(Environment environment, MessageService messageService) {
+        super(Void.class, environment, LOGGER);
+        this.messageService = messageService;
+    }
+
+    @Override
+    protected ResourceMessages[] processInput(Void input, RequestInfo requestInfo, Context context) {
+        String feideId = requestInfo.getFeideId().orElse(null);
```

Comment:
Listing messages only for user for now. Listing messages for Curator will come later